installing/installing_vsphere/installing-vsphere-network-customizations.adoc

```diff
@@ -69,11 +65,9 @@ include::modules/installation-approve-csrs.adoc[leveloffset=+1]
 
 include::modules/installation-operators-config.adoc[leveloffset=+1]
 
-include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
-
-include::modules/registry-configuring-storage-vsphere.adoc[leveloffset=+3]
+include::modules/registry-removed.adoc[leveloffset=+2]
 
-include::modules/installation-registry-storage-non-production.adoc[leveloffset=+3]
+include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
```

Comment:
This module should either only exist in the registry/configuring-registry-storage/configuring-registry-storage-vsphere.adoc assembly or be removed entirely. Is that correct?